### PR TITLE
Finished the project

### PR DESCRIPTION
**Pull request by carloshdelreal**
all the items are responsive and pretty much as the origina webpage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,826 @@
...
+                    <li><a href="">JOBS</a></li>
+                    <li><a href="">CONTACT</a></li>
+                </ul>
+            </div> 
+        </div>
+        <div class="footer-legal">
+            <div class="wrapper">
+                <span>
+                    <a href="#">
+                        <img src="img/tnw.svg"/>
```
Comment (bellom) [inline index.html]: An `img` element must have an alt attribute, except under certain conditions, correct this.

File: index.html [hunk 2]
```diff
@@ -8,6 +8,826 @@
...
+                    </a>
+                    © 2006–2019 The Next Web B.V.
+                </span>
+                <span>
+                    Made with <b>♥</b> in Amsterdam.
+                </span>
+                <span>
+                    Powered by
+                    <a href="#">
+                        <img src="img/stackpath.svg" />
```
Comment (bellom) [inline index.html]: An `img` element must have an alt attribute, except under certain conditions, correct this.

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @carloshdelreal @ViriCruz,
You have done a great job, just few things left to have :+1: , which are;
1. The navbar should be fixed just like the original website.
2. Correct the validation error.
**After fixing the issues please send another request to re-review this project.**

Tips:
For every HTML/CSS projects please check your files with the W3 Validator and the CSS Linter

**Comment by ViriCruz** [comment]
Hi @bellom I was wandering if our project is ready to merge. Let us know please :).

**Comment by bellom** [comment]
Awesome job guys, kindly submit another code review form....

**Review by bellom — APPROVED** [review]
Hi @carloshdelreal @ViriCruz, 
## Congratulation guys :+1:

**Pull request merged**